python-sdk/examples/metadata_rest_api_example.py

```
@@ -0,0 +1,158 @@
+            "user_id": user_id,
+            "thread_id": thread_id,
+            "customer_id": customer_id,
+            "labels": labels or [],
+            # =========================================
+            # Custom metadata — any other keys you want
+            # =========================================
+            **(custom_metadata or {}),
+        },
+    }
+
+    response = requests.post(
```

Ruff (0.15.0)

[error] Probable use of `requests` call without timeout

(S113)